Docs/animation.md

```diff
@@ -0,0 +1,49 @@
+# Animation in Radium
+
+The Radium Engine provides a set of classes dedicated to animation,
+which can be found in `Core/Animation`.
+
+## Animation Basics
+
+An `Animation` is a set of key poses, each associated to a key time.
+When looking for the pose at a specific animation time, 
+the 2 closest surrounding key poses are linearly interpolated to build the querried pose.
+
```

Comment:
Would be nice to have it for both loading animations (if the loader does not perform the interpolation for each animation frame -- which assimp does for example) and also editing / playing animations (which is under development for now -- but only with linear interpolation though).

Reply:
There is a lot of work to be done on the animation framework, to get a proper timeilne and so on.
We have some code already that should be used as basis, but clearly this will require important changes in the code and on the documentation, which is way beyond this PR. I am fine with the current state.

Reply:
I'm fine also with the current state. We can merge.